### PR TITLE
Revise FAQs and README

### LINKED ISSUE #2892: High level tracking of Docs Views with HEAP
## Description

While I was trying to find out where is our Architecture diagram, I found that our FAQ pages view has gone down signficantly.

image

I try to make sense with the data, but I couldn't make HEAP works. I see consistent pattern across configuration, FAQ, data_catalog etc have a view drop in April while we have highest number of unique user of our docs at the same time.

## Context

I would like to have at least high level tracking of important themes, particularly to help me understand what are the drivers of the view change.

This is hard to do because HEAP is not very flexible (or maybe I don't know enough) and the fact we change our domain from readtodoc -> kedro.org which makes it hard to match the statistics 

- Notebook
- Catalog
- Configuration

This could be useful to understand if we are improving our docs. In addition, it may be useful to track specific features adoption.

### PR DESCRIPTION
**Pull request by stichbury**
> **NOTE:** Kedro datasets are moving from `kedro.extras.datasets` to a separate `kedro-datasets` package in
> [`kedro-plugins` repository](https://github.com/kedro-org/kedro-plugins). Any changes to the dataset implementations
> should be done by opening a pull request in that repository.
## Description
Maintenance to answers some discussion on FAQ page usages and revise the links to our Linen slack archive.
fix #2892
## Development notes
This PR makes some minor changes to FAQs to bring it to the top level. It adds links to the website FAQs, and revises the README to add links there too, plus fix some broken links. Removes the companies and awards sections and moves them to `awesome-kedro`.

## Checklist

- [x] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes

### CODE REVIEW

File: docs/source/get_started/install.md [hunk 1]
```diff
@@ -134,7 +134,7 @@ You should see an ASCII art graphic and the Kedro version number. For example:
 
 ![](../meta/images/kedro_graphic.png)
 
-If you do not see the graphic displayed, or have any issues with your installation, check out the [searchable archive of Slack discussions](https://www.linen.dev/s/kedro), or post a new query on the [Slack organisation](https://slack.kedro.org).
+If you do not see the graphic displayed, or have any issues with your installation, check out the [searchable archive of Slack discussions](https://linen-slack.kedro.org/), or post a new query on the [Slack organisation](https://slack.kedro.org).
```
Comment (tynandebold) [inline docs/source/get_started/install.md:137]: A question about writing style: do we prefer `do not`, `don't`, or have no preference? I see both used throughout our docs.

**Review by tynandebold — APPROVED** [review]
Looks great!

I asked one question for my own knowledge. Thank you :)

**Comment by tynandebold** [comment]
Also I would say this closes #2892.

File: docs/source/get_started/install.md (hunk 1 above)
Comment (stichbury) [reply docs/source/get_started/install.md:137]: Good question, no real answer. I'm happy with either and don't have sufficient bandwidth to enforce one or other, even if I had a preference. There's a ticket in this sprint for setting up a language "linter" which we could use to offer consistency one way or the other. But I think life's too short and each communicates perfectly clearly. So I doubt we'll enable that rule.

**Review by merelcht — APPROVED** [review]
LGTM, thanks @stichbury ⭐

File: README.md [hunk 2]
```diff
@@ -78,54 +79,9 @@ Yes! Want to help build Kedro? Check out our [guide to contributing to Kedro](ht
 
 ## Where can I learn more?
 
-There is a growing community around Kedro. Have a look at the [Kedro FAQs](https://docs.kedro.org/en/stable/faq/faq.html#how-can-i-find-out-more-about-kedro) to find projects using Kedro and links to articles, podcasts and talks.
-
-## Who likes Kedro?
-
```
Comment (yetudada) [inline README.md:84]: I think the only thing that's now missing from the README.md is how do contributors know that they can add themselves to this list on `awesome-kedro`?

Comment (stichbury) [reply README.md:84]: Good point, I'll add a sentence.

**Comment by noklam** [comment]
@stichbury ops. it just get merged before I commented.

**Comment by stichbury** [comment]
It was automerged, but anyway, i'm taking your comments on board.

1. Indexing the wiki looks like it is sort-of possible for repos with 500+ stars (us) if we limit wiki editing to repository collaborators (see https://github.com/orgs/community/discussions/4992). The indexing was limited because of github concerns about SEO 🤦 and this is a great resource to find out more https://github-wiki-see.page/

If we don't get the indexing working I'll coordinate with @astrojuanlu and we'll make a subproject for contributor docs to put them somewhere else.

2. Let's put the architecture diagram into a blog post. https://github.com/kedro-org/kedro-devrel/issues/108

**Comment by noklam** [comment]
This sounds good to me! Didn't know about the Github Wiki SEO thing!

**Pull request merged**